### ISSUE TITLE
The Enable method in the Win64 environment cannot be used properly

### ISSUE THREAD

**Issue by hzhappycoder** [soukoku/ntwain#45]
Hello, when I ran the Net5Winform program in the Win64 environment and called the Enable method to open the scanner's UI page, the opened scanner window could not be closed and could not scan properly. But in the Win32 environment, there is no such problem. By the way, the model of the scanner is Cannon C225

**Comment by nikita-ermolaev-coral** [comment]
Do you resolve this error?

**Comment by happycoderNo1** [comment]
> Do you resolve this error?

No, so I have to set the program to 32-bit. Can you tell me what good solution you have?

**Comment by nikita-ermolaev-coral** [comment]
No, I don't have good solution too..
I will trying to find any errors in source code, I need 64 bit application..
Currently I'm trying to scan on Samsung Xpress m2070FW (using default DS's UI). Preview button works fine and scanning is really starts, but Scan button doesn't work at all.
![image](https://github.com/soukoku/ntwain/assets/79456433/37fecea3-5bc9-40aa-b26b-7c1333dce351)
May be your problem was the same?
I'm trying to set up the Sample.WPF example
